scripts/initial

```diff
@@ -0,0 +1,21 @@
+# This section needs to be done manually, and only needs doing after
```

Comment:
So these are the same steps details in the README and specified as 'needs to be done manually' (because of the password setting?). Does that mean this can't be run as a script, need to execute each step individually? If so, probably don't include this at all, since it's documented in the readme,

Reply:
oops